app/bibleview-js/src/components/OSIS/W.vue

```diff
@@ -112,9 +112,13 @@ export default {
   .link-style {
     text-decoration: underline dotted;
   }
-.strongs {
+.skip-offset {
```

Comment:
skip-offset should not be styled. It's markup for highlight code. Better add separate class.

Reply:
I'll fix this after merge.